addons/mail/models/mail_channel.py

```diff
@@ -352,6 +352,11 @@ def add_members(self, partner_ids=None, guest_ids=None, invite_to_rtc_call=False
                     'id': channel_partner.guest_id.id,
                     'name': channel_partner.guest_id.name,
                 })
+                guest = channel_partner.guest_id
+                if guest:
+                    self.env['bus.bus']._sendone(guest, 'mail.channel/joined', {
```

Comment:
improvement (not for this PR), but this method is doing plenty of `_sendone`, it would be nice to build a notification list and only call one `_sendmany` at the end